docs/resources/azure_monitor_log_profile.md.erb

```diff
@@ -0,0 +1,88 @@
+---
+title: About the azure_monitor_log_profile Resource
+platform: azure
+---
+
+# azure\_monitor\_log\_profile
+
+Use the `azure_monitor_log_profile` InSpec audit resource to test properties
+of an Azure Monitor Log Profile.
+
+<br />
+
+## Syntax
+
+An `azure_monitor_log_profile` resource block identifies a Log Profile by name.
+
+    describe azure_monitor_log_profile(name: 'default') do
+      ...
+    end
+<br />
+
+## Examples
+
+### Test that a Log Profile exists
+
+    describe azure_monitor_log_profile(name: 'default') do
+      it { should exist }
+    end
+
+### Test that a Log Profile retention is enabled
+
+    describe azure_monitor_log_profile(name: 'default') do
+      its('retention_enabled') { should be true }
+    end
+
+<br />
+
+## Parameters
+
+* `name`
+
+## Parameter Examples
+
+The name of the Log Profile.
+
+    describe azure_monitor_log_profile(name: 'default') do
+      it { should exist }
+    end
+
+## Properties
+
+* `retention_enabled`
+* `retention_days`
+
+### retention_enabled
+
+Determine if retention is enabled
+
+    its('retention_enabled') { should be true }
+
+### retention_days
+
+Determine number of days retention is enabled for
+
+    its('retention_days')    { should eq(365) }
+
+## Matchers
+
+This InSpec audit resource has the following special matchers. For a full list
+of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### exists
+
+The control will pass if the resource returns a result. Use `should_not` if you expect zero matches.
+
+    # should always exist
+    describe azure_monitor_log_profile(name: 'default') do
+      it { should exist }
+    end
+
+    # should not exist
```

Comment:
This comment pair use neutral. Doesn't explain badly, but doesn't read as contradictory like those above. I'd suggest improving them in the way mentioned already, but it's not entirely necessary here.